streams/do.m3u

```diff
@@ -146,7 +146,7 @@ https://ss3.domint.net:3136/gtv_str/globalhd/playlist.m3u8
 #EXTINF:-1 tvg-id="GuacaraTV.do",Guacara TV (720p)
 https://imagenuniversaltv.net:3820/live/guacaratvlive.m3u8
 #EXTINF:-1 tvg-id="GuaymateTV.do",Guaymate TV (360p)
-https://5790d294af2dc.streamlock.net/8100/8100/playlist.m3u8
```

Comment:
This stream still works. 
Can't tell whether the programing is for Guaymate TV or not.

Reply:
Yes , the link had changed, there is different channels on that Link

Reply:
Whatever that was it's currently shows 404. Not 24/7

Reply:
![image](https://github.com/user-attachments/assets/ccdb78cd-ad18-475a-899b-d2a2c21a9ce8)

Reply:
Many channel front Dominican Republic when off line last week and some then changed they link, you can tell by the logo, that the original channel is on the other link, now the channel that come back on that link after off time was different channel with different logo, I had see this before on this streamlock link

Reply:
I forgot to mention, that link used to be Farovision tv , like 4 years ago, then they changed to Guaymate, i juts look at it on a old playlist that I had saved from 2019.